popup.css

```diff
@@ -83,6 +83,27 @@
   gap: 10px;
 }
 
+.btnn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 7px 0;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background-color: #fff;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 24px;
+  color: #396df2;
+  cursor: pointer;
+}
+
+a {
+  text-decoration: none;
+}
+
```

Comment:
**[OPTIONAL]**
- [ ] It's great that you have implemented the color for both buttons but I would apologize on the behalf of the previous reviewer because he made a mistake about the **See Source** text color because the **See Source** button has the text color of `#6070ff`